client/src/modules/Admin/Styles/Admin.module.css

```diff
@@ -0,0 +1,30 @@
+.semesterUpdate {
+    border: 2px solid #ccc;
+    padding: 10px;
+    border-radius: 5px;
+    margin: 10px;
+    flex: 1;
+}
+
+.semesterButtons {
+    background-color: var(--clr-gray-300);
+    color: var(--clr-white);
+    font-weight: var(--semi-bold-weight);
+    font-size: var(--font-small-size);
+    border-radius: 5px;
+    margin-top: 5px;
```

Comment:
1. looks like this class is for a single button, so should rename to `semesterbutton` (singular) for best practice 
2. instead of uneven margins, consider wrapping all the buttons in a flex display with a set gap attribute 
ie: 
```
.buttonsContainer { 
display: flex;
flex-flow: row wrap;
gap: 5px;
... 

and then give each button a max width as well so it won't take the full row 
}
```